### ISSUE TITLE
Release JFXtras built on JavaFX 17

### ISSUE THREAD

**Issue by DJViking** [JFXtras/jfxtras#131]
There are jfxtra releases for 
JavaFX 11 (11-r2)
JavaFX 14 (14-r2)
JavaFX 15 (15-r2)

Are there any plans to build a release for JavaFX 17 LTS soon?

Also there is no information about these releases on the GitHub page of jfxtras.

**Comment by tbee** [comment]
Yes, there are plans to build for JFX 17, but my other pet project (teslatasks) is having issues, so I'm spread out a bit. But the 11 version (which contains all backports) will run on all 11+.

I stopped mentioning all the versions on the github pages, and in fact am considering only supporting a version based on the LTS.
